src/metatrain/experimental/alchemical_model/default-hypers.yaml

```diff
@@ -26,5 +26,8 @@ architecture:
     log_interval: 5
     checkpoint_interval: 25
     per_structure_targets: []
-    loss_weights: {}
     log_mae: False
+    loss:
+      weights: {}
+      type: mse
```

Comment:
I would switch the type and the weight dictionary.

Reply:
Sure